lib/util/support.js

```diff
@@ -34,6 +34,13 @@ function isSupported() {
     return false;
   }
 
+  // NOTE (joma): Return if iOS chrome version is not 14.3 and above.
+  if (isChromeIOS()) {
+    if (!isChromeVersionSupported()) {
+      return false;
+    }
```

Comment:
So we don't support chrome on ios 14.3 and above?

Reply:
We do support chrome (more specifically WKWebView) on iOS 14.3 and above. If `isChromeVersionSupported` is false, then we will return false. But `getUserMedia` is not available in iOS versions below 14.3. But good point, maybe I should rename this util to something else if it helps make it clearer? How does something like `isiOSVersionSupported` sound? 🤔